kymatio/utils.py

```diff
@@ -0,0 +1,53 @@
+from sklearn.base import BaseEstimator, TransformerMixin
+import torch
+
+
+class ScatteringTransformer(BaseEstimator, TransformerMixin):
+    def __init__(self, S, signal_shape, frontend='numpy'):
+        """Creates an object that is compatible with the scikit-learn API
+        and implements the .transform method.
+        
+        Parameters
+        ==========
+
+        S: an instance of Scattering1D, Scattering2D or Scattering3D
+            This instance is called by the transformer
+
+        signal_shape: tuple of ints
```

Comment:
Can't this be acquired from `S`?

Reply:
your right. But if we are switching to the transformer creating the scattering object this is moot